pkg/controller/batchrelease/workloads/workload_rollout_control_plane.go

```diff
@@ -119,6 +119,10 @@ func (c *UnifiedWorkloadRolloutControlPlane) PrepareBeforeProgress() (bool, erro
 	return true, nil
 }
 
+func (c *UnifiedWorkloadRolloutControlPlane) PrepareOneBatch() (bool, error) {
+	return true, nil
```

Comment:
we can also pause statefulset like workload by setting a very large partition value

Reply:
webhook will do this operation.